pkg/porter/schema.go

```diff
@@ -86,24 +84,39 @@ func (p *Porter) GetManifestSchema() (map[string]interface{}, error) {
 			continue
 		}
 
-		mixinSchemaMap := make(map[string]interface{})
+		// Update relative refs with the new location and reload
```

Comment:
Worth mentioning that this is done to play nicely with older versions of jsonschema?  Perhaps if we have intentions to remove this logic once we're able to update to a more recent jsonschema version.

Reply:
I can't even say why at this point. It's to work around bugs found in VS Code's support for json schema validation which comes from the RH Yaml extension which sounds like it's supported by a college student?

https://github.com/redhat-developer/vscode-yaml/issues/129

They don't know what version of json schema it is, so it may be a bug?

I'm tracking a list of bugs and workarounds [here](https://gist.github.com/carolynvs/c57c3517bd228d4c5b4d0dc24334fe7d). Since it's a lot of conjecture I didn't want to bake accusations into the code though, you know?

Reply:
Sorry not trying to be defensive and I realize that I said it's due to older versions of json schema previously. I've been digging further with Ivan (and just thinking about this way too much), and I change my mind about 20 times a day about what is wrong! 😝